support/environment.go

```diff
@@ -78,6 +79,10 @@ func GetRayImage() string {
 	return lookupEnvOrDefault(CodeFlareTestRayImage, RayImage)
 }
 
+func GetRayAMDGpuImage() string {
```

Comment:
I'd suggest to call it `GetRayROCmImage`, and maybe rename `CodeFlareTestRayImage` to `CodeFlareTestRayCUDAImage`.